src/composite/kron.jl

```diff
@@ -135,7 +135,7 @@ julia> kron(1=>X, 3=>Y)
 Base.kron(blocks::Pair{Int, <:AbstractBlock}...,) = @λ(n->kron(n, blocks...))
 Base.kron(blocks::Base.Generator) = kron(blocks...)
 
-occupied_locs(k::KronBlock) = Iterators.flatten(map(x-> x + i - 1, occupied_locs(b)) for (i, b) in zip(k.locs, subblocks(k)))
+occupied_locs(k::KronBlock) = (Iterators.flatten(map(x-> x + i - 1, occupied_locs(b)) for (i, b) in zip(k.locs, subblocks(k)))...,)
```

Comment:
```suggestion
occupied_locs(k::KronBlock) = Tuple(Iterators.flatten(map(x-> x + i - 1, occupied_locs(b)) for (i, b) in zip(k.locs, subblocks(k))))
```

`Tuple` is more explicit. but not a big deal.

Reply:
`Tuple` is not good, Andreas showed me some examples that `Tuple` is very slow and is sometimes not type stable.
```julia
julia> @benchmark ($a...,)
BenchmarkTools.Trial: 
  memory estimate:  32 bytes
  allocs estimate:  1
  --------------
  minimum time:     265.784 ns (0.00% GC)
  median time:      296.468 ns (0.00% GC)
  mean time:        327.817 ns (7.42% GC)
  maximum time:     171.090 μs (99.79% GC)
  --------------
  samples:          10000
  evals/sample:     264

julia> @benchmark Tuple(i for i in $a)
BenchmarkTools.Trial: 
  memory estimate:  160 bytes
  allocs estimate:  3
  --------------
  minimum time:     330.805 ns (0.00% GC)
  median time:      387.059 ns (0.00% GC)
  mean time:        462.967 ns (9.13% GC)
  maximum time:     220.214 μs (99.78% GC)
  --------------
  samples:          10000
  evals/sample:     205
```